styles/styles.css

```diff
@@ -232,10 +232,16 @@ main li {
   list-style: none;
 }
 
+main .section.column-with-list.navy-background.columns-container {
```

Comment:
is these changes are related to the marketing pages, then ideally you should move this to the marketing template css

Reply:
these are changes to column-with-list!